types.ts

```diff
@@ -155,6 +155,9 @@ export interface OAuth2Authentication extends BaseAuthentication {
 
   // Some OAuth providers require passing the token as a URL param.
   tokenQueryParam?: string;
+
+  // The charactor used to separate multiple OAuth scopes. Default is a whitespace.
```

Comment:
Nit, would say `Default is a single space, per the OAuth standard.` Would use a `/** */` style comment while you're here so it makes it into TypeDoc (we'll eventually go clean up the rest of them).